resources/ext/ezbake.conf

```diff
@@ -2,8 +2,8 @@ ezbake: {
   pe: {}
   foss: {
     redhat: { dependencies: ["puppet >= 3.7.3", "puppet < 5.0.0"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] },
-    debian: { dependencies: ["puppet (>= 3.7.3-1puppetlabs1)", "puppet (<< 5.0.0-1puppetlabs1)"],
-              postinst: ["/usr/bin/puppetdb ssl-setup"] }
+              postinst: ["PATH=/opt/puppetlabs/bin:/usr/bin:$PATH puppetdb ssl-setup"] },
```

Comment:
@kbarber My reasoning here was that if the user has FOSS puppet 3.x installed, they will have the puppet binaries installed in /usr/bin. Since 3.x compatibility is desired, this helps enable that (in the super unlikely case that they don't have /usr/bin in PATH). I could be totally mistaken here of course.